index.rst

```diff
@@ -174,14 +174,14 @@
             <div class="col-sm-4 front-block-text">
                <h2>News</h2>
 
-               <ul>
+               <ul>            
```

Comment:
```suggestion
               <ul>            
```

Reply:
How to commit this suggestion @weiji14 ?

Reply:
Oops, forgot to delete the blankspace. Should be ok now.